src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt

```diff
@@ -0,0 +1,72 @@
+package me.zeroeightsix.kami.feature.module
+
+import baritone.api.BaritoneAPI
+import baritone.api.process.IBaritoneProcess
+import baritone.api.process.PathingCommand
+import baritone.api.process.PathingCommandType
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.BaritoneIntegration
+import me.zeroeightsix.kami.KamiMod
+import me.zeroeightsix.kami.event.TickEvent
+
+@Module.Info(
+    name = "BaritonePause",
+    category = Module.Category.MISC,
+    description = "Pauses Baritone"
+)
+object BaritonePause : Module() {
+    var process: BaritoneWrapper? = null
+    var registeredProcess = false
+
+    init {
+        hidden = true
+        BaritoneIntegration {
+            hidden = false
+            process = BaritoneWrapper(BaritoneWrapper.PauseBaritoneProcess("${KamiMod.MODNAME} Pause Module"))
+        }
+    }
+
+    @EventHandler
+    val onTick = Listener<TickEvent.InGame>({
+        tryInitProcess()
+    })
+
+    private fun tryInitProcess() {
+        if (registeredProcess) return
+        BaritoneIntegration {
+            val mngr = BaritoneAPI.getProvider()?.primaryBaritone?.pathingControlManager
+            mngr?.let {
+                it.registerProcess(process?.process)
+                registeredProcess = true
+            }
+        }
+    }
```

Comment:
This doesn't sit right with me. It's rather yucky to
1. maintain state with `registeredProcess`
2. have a tick listener that will basically do nothing except for 1 tick

I'd rather have you subscribe a one-off listener that unsubscribes itself:

```kt
BaritoneIntegration {
    hidden = false
    process = BaritoneWrapper(BaritoneWrapper.PauseBaritoneProcess("${KamiMod.MODNAME} Pause Module"))
    var listener: Listener<TickEvent.InGame>? = null
    listener = Listener({
        val manager = BaritoneAPI.getProvider()?.primaryBaritone?.pathingControlManager
        manager?.let {
            it.registerProcess(process?.process)
            KamiMod.EVENT_BUS.unsubscribe(listener)
        }
    })
    KamiMod.EVENT_BUS.subscribe(listener)
}
```
This way there is no need for the tick listener or `tryInitProcess`